staging/src/k8s.io/kubectl/pkg/cmd/delete/delete.go

```diff
@@ -333,25 +346,24 @@ func (o *DeleteOptions) DeleteResult(r *resource.Result) error {
 		}
 		if status, ok := response.(*metav1.Status); ok && status.Details != nil {
 			uidMap[resourceLocation] = status.Details.UID
-			return nil
+			continue
 		}
 		responseMetadata, err := meta.Accessor(response)
 		if err != nil {
 			// we don't have UID, but we didn't fail the delete, next best thing is just skipping the UID
 			klog.V(1).Info(err)
-			return nil
+			continue
 		}
 		uidMap[resourceLocation] = responseMetadata.GetUID()
-
-		return nil
-	})
-	if err != nil {
-		return err
 	}
-	if found == 0 {
-		fmt.Fprintf(o.Out, "No resources found\n")
-		return nil
+
+	if len(errs) > 0 {
```

Comment:
This is the same mechanism how `ContinueOnErrorVisitor` does https://github.com/kubernetes/kubernetes/blob/a5aa858d44651ba48bdce634a39b55be91216614/staging/src/k8s.io/cli-runtime/pkg/resource/visitor.go#L372